### DISCUSSION TITLE
Does anyone use Autogluon TimeSeries to predict wind power?

### DISCUSSION THREAD

**Discussion by isCopyman** [autogluon/autogluon#4084] [Q&A]
https://www.kaggle.com/datasets/berkerisen/wind-turbine-scada-dataset
It seems that Autogluon TimeSeries does not perform well on wind power prediction datasets

**Comment by shchur** [comment]
Hi @isCopyman, thank you for sharing this dataset and problem setting. Are you aware of some methods that perform well on this type of time series data? We could consider including them to the AutoGluon model zoo to better handle this application domain.
